### PR TITLE
Remove CSP header for admin views which was broken Flask-Admin forms

### LINKED ISSUE #56: Content-Security-Policy header breaks Flask-Admin forms
The Talisman extension in Invenio app module adds the security header
`[X-]Content-Security-Policy:default-src 'self'`

This breaks the Flask-Admin forms because the code has inline js/css.

There is an issues opened on Flask-Admin and a PR partially fixing the issue (https://github.com/flask-admin/flask-admin/issues/1135). For the time being, the easiest solution is to remove the header for Invenio admin views.

### PR DESCRIPTION
**Pull request by ntarocco**
- override Talisman configuration @before_request to set CSP to None
- remove CSP response header
- fixes #56

### CODE REVIEW

**Comment by ntarocco** [comment]
@lnielsen and @diegodelemos if you want to avoid the extra attribute is_admin in the ProtectedAdminView class, another solution (less/more clean?) is to move out from the factory the ProtectedAdminView and use `isinstanceof(view_function.im_class, ProtectedAdminView)` in the before_request/after_request.

File: invenio_admin/ext.py [hunk 1]
```diff
@@ -166,6 +169,42 @@ def lazy_base_template():
...
+        @app.before_request
+        def _disable_talisman_csp_admin_views():
+            """Override Talisman configuration for admin views by setting the CSP to None before the request.
+            There is no certitude that this @before_request will be executed before Talisman @before_request: this
+            case is covered and fixed by the @after_request.
+            """
+            view_function = current_app.view_functions.get(request.endpoint)
+            if _is_protected_admin_view(view_function):
+                invenio_app = app.extensions['invenio-app']
+                setattr(invenio_app.talisman.local_options, 'content_security_policy', None)
```
Comment (slint) [inline invenio_admin/ext.py]: Shouldn't this be done in `after_request`? At this point it's not clear if talisman's `before_request` has been applied or not. On the other hand in `after_request` it's certain that the `talisman.local_options` object exists.

Another thing is that this patch assumes that `invenio-app` is being used (which might not be always the case), so probably an `if` clause might be necessary. @lnielsen, is there a plan to use `invenio-app` by default with the `app` fixture in `conftest.py` of all Invenio modules?

**Comment by ntarocco** [comment]
I have updated the PR with the changes checked together. Fixed a typo (Administration) and also some auto-formatting.
Let me know if you "don't want to see" the autoformatting noise, or you actually prefer to fix the formatting.

**Comment by lnielsen** [comment]
Re. the formatting the it depends on size. Fixing some few formatting issue is fine in the same commit, fixing a lot of formatting is better in separate commit to make clear what is feature/bug fix vs  style fixes. Can you check your line length configuration, some lines seems too long (our default is 80). Otherwise looks good to me.

**Comment by ntarocco** [comment]
@lnielsen I have changed the max line length value, and pushed again. Thanks for checking!

**Pull request merged**